### PR TITLE
unzip: Depends on bzip2 and fixes for Linuxbrew

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: unzip.rb [hunk 1]
```diff
@@ -43,13 +45,15 @@ class Unzip < Formula
...
+    args = []
+    args << "LFLAGS1=-liconv" if OS.mac?
     system "make", "-f", "unix/Makefile",
       "CC=#{ENV.cc}",
       "LOC=-DLARGE_FILE_SUPPORT",
       "D_USE_BZ2=-DUSE_BZIP2",
       "L_BZ2=-lbz2",
-      "LFLAGS1=-liconv",
-      "macosx"
+      "macosx",
```
Comment (DomT4) [inline unzip.rb]: Is `macosx` not problematic for Linux, or?

Comment (sjackman) [inline unzip.rb]: Believe it or not, the `macosx` build rule is actually much saner than the `linux` build rule, which is weird. In fact, that whole Makefile is one of the more odd Makefiles that I've ever seen in my life. Short answer though, the `macosx` target works just fine on Linux.

Comment (DomT4) [inline unzip.rb]: Heh, oh well 🙈. Thanks for the answer!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-dupes/commit/9e4a084bdcfa435c5a00e6b5df4961c1809c988a. Thanks @sjackman!

**Comment by sjackman** [comment]
Thanks for the merge, Dominyk!

**Pull request merged**